rust/utils/run-locally/src/invariants.rs

```diff
@@ -87,11 +98,14 @@ pub fn termination_invariants_met(
     .iter()
     .sum::<u32>();
     // The relayer and scraper should have the same number of gas payments.
-    if gas_payments_scraped != gas_payment_events_count {
+    // TODO: Sealevel gas payments are not yet included in the event count.
```

Comment:
hmm why is this the case?

Reply:
I think they're just counted separately in prometheus metrics?